app/code/Magento/Review/Model/Review/AddRatingVotesToCustomerReviews.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Review\Model\Review;
+
+use Magento\Review\Model\ResourceModel\Rating\Option\Vote\Collection as OptionVoteCollection;
+use Magento\Review\Model\ResourceModel\Rating\Option\Vote\CollectionFactory as OptionVoteCollectionFactory;
+use Magento\Review\Model\ResourceModel\Review\Product\Collection;
+
+/**
+ * The model that adds the rating votes to reviews
+ */
+class AddRatingVotesToCustomerReviews
+{
+    /**
+     * @var RatingOptionCollectionFactory
+     */
+    private $ratingOptionCollectionFactory;
+
+    /**
+     * @param OptionVoteCollectionFactory $ratingOptionCollectionFactory
+     */
+    public function __construct(OptionVoteCollectionFactory $ratingOptionCollectionFactory)
+    {
+        $this->ratingOptionCollectionFactory = $ratingOptionCollectionFactory;
+    }
+
+    /**
+     * Add rating votes to customer reviews
+     *
+     * @param Collection $collection
+     */
+    public function execute(Collection $collection): void
+    {
+        $connection = $collection->getConnection();
```

Comment:
In this logic extracted from some other method? If yes, please make sure we do not have code duplicating and reuse this class in place from where it is extracted.

Reply:
I've refactored a bit the implementation, and this model isn't needed anymore.